app/models/katello/glue/candlepin/product.rb

```diff
@@ -63,11 +71,12 @@ def self.import_product_content(product, content_attrs)
           content_url: pc[:contentUrl]
         }
 
-        # current product has this content - update it
+        # current product has this content - update its name if needed
         # otherwise create a reference to existing content OR new content altogether
         if (existing = product.product_content_by_id(pc[:id]))
-          existing.content.update_attributes!(content_attrs)
-          existing.update_attributes(enabled: params['enabled'])
+          new_name = pc['name']
+          existing.content.update_attributes!(:name => new_name) if existing.content.name != new_name
+          existing.update_attributes(enabled: params['enabled']) if existing.enabled != params['enabled']
```

Comment:
You want to ignore if this one fails? First non-! update I've seen in this PR. ;)